spec/models/tenant_spec.rb

```diff
@@ -520,6 +520,7 @@
           :vms_allocated => {
               :unit          => "fixnum",
               :value         => 20.0,
+              :warn_value    => nil,
               :format        => "general_number_precision_0",
               :text_modifier => "Count",
               :description   => "Allocated Number of Virtual Machines"
```

Comment:
Not for this PR, but these tests should be checking for 20 and 4096 and maybe that the type matches what we expect.  warn_value is tested in the other spec. The extra key/values in the test are extra information that will likely break when we change things going forward.